DietPi-Software | Portainer enhancement

Joulinar opened the pull request:
Portainer images using different names for 1.x and 2.0. Therefore we would need to use `wildcards` to detect all image version

**Status**:  Ready 
- [x] check re-installation and upgrade from 1.x to 2

**Reference**: https://github.com/MichaIng/DietPi-Docs/pull/301

**Commit list/description**:
- DietPi-Software | Portainer: ensure that all image version of Portainer are detected during installation or removal

Commit:
v6.35

+ DietPi-Software | Portainer: ensure that all image version of Portainer are detected during installation or removal

MichaIng commented:
Portainer 2.0 was released already when we started to implement it, so actually supporting `portainer-ce` only should be sufficient? Although one could have installed Portainer previously and now wants to overwrite it by the DietPi implementation.

The only concern I have is that there may be other containers that match the wildcard, e.g. some extensions or companion containers? The following would match both as well, right?
```
docker ps -a | mawk '/portainer\/portainer(-ce)?( |$)/{print $1}'
docker images -a | mawk '/portainer\/portainer(-ce)?( |$)/{print $3}'
```

Commit:
v6.35

DietPi-Software | Portainer: reduce image detection to Portainer 1 and 2, to avoid wildcards.

Joulinar commented:
> Although one could have installed Portainer previously and now wants to overwrite it by the DietPi implementation.

Indeed that was the aim to allow transition from v1 to v2. I thought it make sense even if it might be a rar case.

> The only concern I have is that there may be other containers that match the wildcard, e.g. some extensions or companion containers?

You are right, better to stay with dedicated container names and to avoid wildcards. Corrected.

Joulinar commented:
for the records. Successfully updated my Portainer from v2.0.0 > v2.0.1 using `dietpi-software reinstall 185`
